#!/usr/bin/env python
PACKAGE = "aerial_robot_base"

from dynamic_reconfigure.parameter_generator_catkin import *

KALMAN_FILTER_FLAG = 0
ACC_SIGMA = 1
BIAS_SIGMA = 2
MEASURE_SIGMA = 3

gen = ParameterGenerator()

gen.add("kalman_filter_flag", bool_t, KALMAN_FILTER_FLAG, "Kalman Filter Flag", False)
gen.add("acc_sigma", double_t, ACC_SIGMA, "Acc Sigma", 0.05, 0, 1)
gen.add("bias_sigma", double_t, BIAS_SIGMA, "Roll/Pitch Bias Sigma", 0.001, 0, 1)
gen.add("measure_sigma", double_t, MEASURE_SIGMA, "Measure Sigma", 0.05, 0, 1)

exit(gen.generate(PACKAGE, "kalman_filter", "KalmanFilterXYBias"))


